domain/src/main/java/com/andersen/dao/availability/AvailabilityDAOImpl.java

```diff
@@ -0,0 +1,114 @@
+package com.andersen.dao.availability;
+
+import com.andersen.connection.DatabaseConnection;
+
+import com.andersen.entity.workspace.Availability;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class AvailabilityDAOImpl implements AvailabilityDAO {
+    private static final Logger logger = LoggerFactory.getLogger(AvailabilityDAOImpl.class);
+
+    @Override
+    public void createAvailability(Availability availability) throws SQLException {
+        String sql = "INSERT INTO availabilities (workspace_id, date, time, capacity, remaining) VALUES (?, ?, ?, ?, ?)";
+        try (Connection connection = DatabaseConnection.getDataSource().getConnection();
+             PreparedStatement statement = connection.prepareStatement(sql)) {
+
+            statement.setLong(1, availability.getId());
```

Comment:
I asked you several times, what is Availability and what do we need it for? Cause i on't understand